specification/apimanagement/resource-manager/readme.enable-multi-api.md

```diff
@@ -0,0 +1,102 @@
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimloggers.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimnotifications.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimnetworkstatus.json
+  - >-
+    Microsoft.ApiManagement/preview/2018-06-01-preview/apimopenidconnectproviders.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimportalsettings.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimproducts.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimproperties.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimquotas.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimreports.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimsubscriptions.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimtagresources.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimtags.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimtenant.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimusers.json
+  - Microsoft.ApiManagement/preview/2018-06-01-preview/apimversionsets.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimanagement.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimapis.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimauthorizationservers.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimbackends.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimcertificates.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimdeployment.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimdiagnostics.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimemailtemplate.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimgroups.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimidentityprovider.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimloggers.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimnotifications.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimnetworkstatus.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimopenidconnectproviders.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimportalsettings.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimproducts.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimproperties.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimquotas.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimreports.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimsubscriptions.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimtagresources.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimtags.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimtenant.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimusers.json
+  - Microsoft.ApiManagement/stable/2018-01-01/apimversionsets.json
+  - Microsoft.ApiManagement/stable/2017-03-01/apimanagement.json
+  - Microsoft.ApiManagement/stable/2017-03-01/apimapis.json
```

Comment:
We will not be generating SDKs or documentation for api-version less than 2018-01-01. We have announced deprecation https://azure.microsoft.com/en-us/updates/azure-api-management-update-oct-18/

Reply:
Ah, well, that's fine and all, but we need to include everything from every version, so that Resource Schemas get generated correctly.